### ISSUE TITLE
Controller Bug

### ISSUE THREAD

**Issue by Charasyte** [bunfan/beat-banger-public#299]
### Discord Tag

@thehonorableoutlaw

### Issue Description

Hi. To be brief, when I connect my controller (an Xbox one controller) to the game, the screen will turn grey, the music will become muffled, and it'll keep saying "missed" as if I were missing the beats. Not sure if it matters but, it doesn't take lives away if you don't mess up after the first beat. The first beat takes a life away but if you keep playing and do it right, although it says your missing, you can keep playing. If you do actually mess up although then you'll lose lives like normal. Just thought it was report worthy. Thanks!

TLDR - Bugged so controller doesn't work

### OS

Windows

### Version

3.49

**Comment by hackerTiger21** [comment]
What exact model of controller are you using, and are you connecting over Bluetooth or USB?

**Comment by hackerTiger21** [comment]
Just as I thought, this is a known bug in the Godot engine. While connected wirelessly, your controller is being recognized as both an Xbox One S controller AND a generic XInput device, thus making a single action result in a double input. This is because the Bluetooth hardware IDs for your controller got changed after a firmware update and never got added to Godot's exception checker list.
